man: document CPUAffinity= in system.conf in more detail

poettering opened the pull request:
Fixes: #9692

keszybz commented:
Looks good, but the dir-locals patch got merged elsewhere. Can you rebase so we can have clean history?

poettering commented:
Thanks for the review. Rebased and dropped the patch. Setting the green label, given that these are the only changes and you already set the blue label. Thanks!
